### PR TITLE
Api rewrite2

### PR DESCRIPTION
**Pull request by kilpatty**
Moving HNScan to Urkel APIs

### CODE REVIEW

File: src/api/blocks.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const { getUrkel, getClient } = require("../util/clients.js");
+
+const { checkUrkel } = require("../util/util.js");
+
+const { getBlock } = require("./index.js");
+
+async function getBlocks(limit = 25, offset = 0) {
+  if (checkUrkel()) {
```
[the remark this answers is not in the document]
Comment (kilpatty) [reply src/api/blocks.js]: ooh not a bad idea I like that brb fixing that lol

**Comment by JGeiser9** [comment]
Status update on this PR?

**Comment by kilpatty** [comment]
Quite a bit aways -> Need to finish hooking this up with Urkel APIs, once we have Nameian in a better place, I'll crank this out.

**Comment by kilpatty** [comment]
This is coming tonight I promise :) @shanejann @JGeiser9

**Pull request merged**